### ISSUE TITLE
[FEA] DataFrame.fillna

### ISSUE THREAD

**Issue by mrocklin** [rapidsai/cudf#1106]
It would be nice to have a `DataFrame.fillna` method

```python
import cudf
df = cudf.DataFrame({'x': [1, 2, None]})
df.fillna(999)
```

Presumably this could just create a new dataframe after having called `.fillna` on all of the series.

cc @beckernick 

Reported downstream in https://github.com/rapidsai/dask-cudf/issues/117

**Comment by mrocklin** [comment]
Also, dask-cudf will probably want the same keywords (and lack of support) as in #1033

**Comment by kkraus14** [comment]
@shwina This could possibly be tackled in #1033 as well?

**Comment by shwina** [comment]
@kkraus14 Yes - I'll include this in #1033

**Comment by shwina** [comment]
@mrocklin To be clear, which keywords specifically are required?

**Comment by mrocklin** [comment]
Ah, I meant to refer to https://github.com/rapidsai/cudf/pull/1107/files
before.  The method, limit, and axis keywords in particular, though I
really only care that the default values work.

**Comment by shwina** [comment]
Thanks, @mrocklin - I'll work on getting those implemented

**Comment by shwina** [comment]
@kkraus14 @mrocklin 

#1033 is ready for review

**Comment by kkraus14** [comment]
Resolved by #1033
